### ISSUE TITLE
请问有做过ASF-YOLO的tensorrt的推理加速吗

### ISSUE THREAD

**Issue by yuzhongx** [mkang315/ASF-YOLO#11]
请问有做过ASF-YOLO的tensorrt的推理加速吗

**Comment by mkang315** [comment]
No, I didn't try using TensorRT acceleration. Please refer to: [Accelerating Inference Up to 6x Faster in PyTorch with Torch-TensorRT](https://developer.nvidia.com/blog/accelerating-inference-up-to-6x-faster-in-pytorch-with-torch-tensorrt).

**Comment by yxl23** [comment]
> 请问有做过ASF-YOLO的tensorrt的推理加速吗

哥们儿，你这个tensorrt弄完了吗

**Comment by mkang315** [comment]
I'm sorry, I didn't actually do TensorRT acceleration.
